bundle/language_zh_CN.properties

```diff
@@ -1,1426 +1,1493 @@
+##############################################
+# Format of this file:
+#
+#  1. Common dialog titles
+#  2. Common button text
+#  3. Common messages, text, and labels
+#  4. Module specific titles (in directory order)
+#  5. Module specific button text (in directory order)
+#  6. Module specific column headers for browser modules (in directory order)
+#  7. Module specific messages and text
+#  8. Text found in CORE repository
+#  9. Remaining original key/values that need review/work/etc.
+# 10. Key/values used in sql files (not changing them at this time to simplify the task)
+##########################################################
 
-angal.admission.abortdate                                                        = éåºæ¥æ
-angal.admission.address                                                          = å°å
-angal.admission.addressm                                                         = å°å
-angal.admission.admitted                                                         = æ¶æ²»
```

Comment:
was the original file not encoded properly (UTF-8)?

Reply:
Yes; that seems to be the issue.